cmd/manager/main.go

```diff
@@ -162,27 +151,3 @@ func main() {
 		os.Exit(1)
 	}
 }
-
-// hasKappApis checks whether the cluster has Kapp APIs installed in the cluster.
-// This does not guarantee that the controller is present to reconcile the App CRs.
-func hasKappApis(config *rest.Config) (bool, error) {
-	discoveryClient, err := discovery.NewDiscoveryClientForConfig(config)
-	if err != nil {
-		return false, fmt.Errorf("creating discovery client: %v", err)
-	}
-	apiResourceList, err := discoveryClient.ServerResourcesForGroupVersion(carvelv1alpha1.SchemeGroupVersion.String())
-	if err != nil && !errors.IsNotFound(err) {
-		return false, fmt.Errorf("listing resource APIs: %v", err)
-	}
-
-	if apiResourceList == nil {
-		return false, nil
-	}
-
-	for _, resource := range apiResourceList.APIResources {
-		if resource.Kind == "App" {
-			return true, nil
-		}
-	}
-	return false, nil
-}
```

Comment:
Why?

Reply:
This change was taken from #690 , in which Varsha explained that the check is not needed due to the feature gate.

Reply:
So, why is it in yours? 😕

Reply:
That PR is undergoing a refactor to combine the tests into a test loop, which is likely going to take some time given its relative priority. This particular change is valuable though since it made doing these tests much easier.